data/lisboa_viva/lines.csv

```diff
@@ -1,8 +1,8 @@
-id,name
-0x021001,Blue
-0x022002,Yellow
-0x023003,Green
-0x024004,Red
+id,name,local_name
```

Comment:
You'll need to add flags to `csv2pb.py` call for these translations to be shown when using Portuguese:

```
-L pt,pt-rPT,pt-rBR -l pt
```

Reply:
Added it and switched to GTFS